pulsar-functions/instance/pom.xml

```diff
@@ -77,6 +77,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
Cleanup, make it use specific version. Comment why we're even pulling this in explicitly.

Reply:
We should stick with single version of Netty. BK 4.9 has broken down from netty-all to individual jars already. We'll need to do it here as well and move to latest netty.